apps/service_gather/lib/gather/extraction/store.ex

```diff
@@ -21,6 +21,19 @@ defmodule Gather.Extraction.Store do
     end
   end
 
+  @spec mark_done(Extract.t()) :: :ok
+  def mark_done(%Extract{} = extract) do
+    Brook.ViewState.merge(@collection, identifier(extract), %{"done" => true})
```

Comment:
Why marking it done vs removing it from the collection?

Reply:
Per Conversation with Johnson. When you go to delete a dataset, the app often needs information in the dataset-state to know for instance, what kafka topic or DB table to delete. If we delete the state at a premature point, we no longer have that needed state.